org.linux_hardware.hw-probe.yaml

```diff
@@ -0,0 +1,436 @@
+app-id: org.linux_hardware.hw-probe
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
```

Comment:
It's a string, needs a double quote for the runtime version

Reply:
Done. Thank you.